src/bin/ion/commands/inspect.rs

```diff
@@ -320,14 +329,14 @@ impl<'a, 'b> IonInspector<'a, 'b> {
                 InspectorAction::LimitReached => break,
             }
 
-            if !is_first_item && !is_last_item && !expr.is_ephemeral() {
+            if !is_first_item && !expr.is_ephemeral() {
                 // If this item is neither the first nor last in the stream, print a row separator.
                 write!(self.output, "{ROW_SEPARATOR}")?;
             }
 
             match expr {
                 EExp(eexp) => {
-                    self.inspect_eexp(0, eexp)?;
+                    self.inspect_eexp(0, "", eexp)?;
```

Comment:
🪧 E-expressions and arg groups now correctly print their trailing delimiter when they appear inside a container.